doc/book/src/basics/one-time-witness.md

```diff
@@ -0,0 +1,31 @@
+# One Time Witness
+
+One Time Witness (OTW) is a special instance of a type which is created only in the module initializer and is guaranteed to be unique and one. It is important for cases where we need to make sure that a witness-authorized action was performed only once (for example - [creating a new Coin](/samples/coin.md)). In Sui Move a type is considered an OTW if it has the following properties:
```

Comment:
```suggestion
One Time Witness (OTW) is a special instance of a type which is created only in the module initializer and is guaranteed to be unique and have only one instance. It is important for cases where we need to make sure that a witness-authorized action was performed only once (for example - [creating a new Coin](/samples/coin.md)). In Sui Move a type is considered an OTW if its definition has the following properties:
```

The second suggestion comes from the fact that the type itself must have more properties (e.g. cannot be explicitly packed) but indeed if its definition has these two properties, it will be considered OTW (unless other properties are violated in which case the whole module will be rejected). In any case, feel free to rephrase as you see fit.